tests/ipex/test_modeling.py

```diff
@@ -44,6 +43,7 @@
     IPEXModelForSequenceClassification,
     IPEXModelForTokenClassification,
 )
+from optimum.intel.openvino.utils import TemporaryDirectory
```

Comment:
you're importing optimum.intel.openvino.utils in ipex paths, no need to change anything in ipex paths if I understand correctly.

Reply:
it was commited accidentally, sorry